contrib/cmake/build3rdparty.cmd

```diff
@@ -12,38 +13,58 @@ set OVERLAYTRIPLETS=--overlay-triplets=%DIR%vcpkg_extra_triplets
  else (if /I "%1" == "-d" (set DEPS_FILE=%2 & shift )^
  else (if /I "%1" == "-p" (set PORTS_FILE=%2 & shift )^
  else (if /I "%1" == "-t" (set OVERLAYTRIPLETS=--overlay-triplets=%2 & shift )^
+ else (if /I "%1" == "-i" (set INPLACE=true )^
```

Comment:
we probably want to have the flag the other way around: do not copy the overlaid ports by default. i.e. keep it working as default the way it was before